src/AcronymLike.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import Card, { CardActions, CardHeader, CardContent } from 'material-ui/Card';
-import Chip from 'material-ui/Chip';
+// import PropTypes from 'prop-types';
```

Comment:
Do we need to keep all these commented imports?

Reply:
I'm ok with removing them. I wasn't sure if someone else had plans for them.